example/app.js

```diff
@@ -19,3 +23,6 @@ module.exports = async function (fastify, opts) {
     options: Object.assign({}, opts)
   })
 }
+
+app[Symbol.for('fastify.display-name')] = 'App Name'
```

Comment:
what's this?

Reply:
`fastify-overview` uses [`fastifyInstance.pluginName`](https://www.fastify.io/docs/latest/Reference/Server/#pluginname) under the hood.

That symbol defines the context's name, replacing the file name.

This example benefits a more preciserer visualization thanks to wording nodes.

Reply:
shall we not use a public API to define this? it's weird to rely on such internals

Reply:
It is used by the `fastify-plugin` https://github.com/fastify/fastify-plugin/blob/3a04d1a868c1030cfe8b32e72292a12271672b08/plugin.js#L48

It should be documented here as the `skip-override` one:
https://www.fastify.io/docs/latest/Reference/Plugins/#handle-the-scope

I will fix it on the fastify repo

Reply:
sounds good thanks

Reply:
Done here https://github.com/fastify/fastify/pull/4191